src/test/resources/org/kohsuke/github/AppTest/wiremock/notifications/mappings/notifications-10-a97934.json

```diff
@@ -2,7 +2,7 @@
   "id": "a979348d-c6be-4cb7-8877-7c42a6f013ae",
   "name": "notifications",
   "request": {
-    "url": "/notifications?all=true&page=9&all=true",
```

Comment:
And by switching to one consistent path we find little bugs like this.  Yay!

Reply:
:)